docs/patterns/tools/use-tools/use-specific-actions.mdx

```diff
@@ -10,112 +10,124 @@ description: "Each Tool (like GitHub, Slack, etc.) comes with many Actions. You
 </Tip>
 
 ### Specifying Actions
+
 `GITHUB_CREATE_AN_ISSUE` and `GITHUB_COMMIT_EVENT` are action IDs for actions in the GitHub Tool
+
 <CodeGroup>
-```python Python
-from composio import ComposioToolSet, App, action
+  ```python Python
+  from composio_langchain import ComposioToolSet, Action
```

Comment:
There's an inconsistency in import statements across examples. Line 18 uses `composio_langchain` while later examples (line 98) use `composio_openai`. Consider using consistent imports across examples or explain why different imports are used.